app/controllers/task_controller.rb

```diff
@@ -1,13 +1,15 @@
 class TaskController < ApplicationController
```

Comment:
Bleargh, you're right. I changed everything over to TasksController and fixed the routes.